app/models/controlled_work_document_content.rb

```diff
@@ -64,33 +66,33 @@ def main_home_percentage_owned
     end
   end
 
-  def additional_non_smod_properties_value
-    additional_non_smod_properties_sum "value"
+  def additional_properties_value(*modifier)
```

Comment:
I don't think you need the `*` here and below - these methods will only ever be called with 1 argument and possibly should explode if we ever try to call them with more.

Reply:
@patrick-laa I ended up adding the `*` as I haven't actually passed in a modifier in the `cw2.yml`. My thinking was, we set a default of `property_type = "all"` so it wasn't necessary to set that in the `cw2.yml` as well - but maybe we should. I didn't want to put `modifier: all` in a bunch of places, but maybe that's the better thing to do. What do you think?

Reply:
@HettieS ah, I see. I agree, we don't want to explicitly add "modifier" lines in the YAML. So if `additional_properties_value` is either going to be called with one or zero arguments, my instinct would be to set the method to `def additional_properties_value(property_type = "all")`, but TBH I do now see why you used `*` and if you wanted to stick with that I think it would be fine.